### PR TITLE
[RM Ch07] Restructuring 7.4.1.2 and 7.4.2

### LINKED ISSUE #1981: [RM Ch07] Restructuring 7.4.1.2 and 7.4.2
Restructuring 7.4.1.2 in order to remove the duplicated items in 7.4.2

### PR DESCRIPTION
**Pull request by karinesevilla**
Fixes #1981 
7.4.1.2 Requirements sorted by category
7.4.2 Duplicated requirements, covered in 7.4.1.2, deleted

### CODE REVIEW

File: doc/ref_model/chapters/chapter07.md [hunk 1]
```diff
@@ -123,44 +123,46 @@ The platform supports the workload, and in effect controls access to the workloa
...
-* Data Centre Operations staff and systems must use management protocols that limit security risk such as SNMPv3, SSH v2, ICMP, NTP, syslog, and TLS.
+**Platform provisioning and LCM**
 * A platform change management process that is documented, well communicated to staff and tenants, and rigourously followed.
 * A process to check change management adherence that is implemented, and rigourously followed.
-* Processes for managing platform access control filters that are documented, followed, and monitored.
-* No login to root on any platform systems, when root privileges are not required.
-* Role-Based Access Control (RBAC) must apply for all systems access.
 * An approved system or process for last resort access must exist for the platform.
-* All back-end API access must use TLS.
+* Where there are multiple hosting facilities used in provision of the service, network communications between facilities for the purpose of backup, management, and application communication are cryptographically protected in transit between data centre facilities.
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter07.md:159]:
```suggestion
* Where there are multiple hosting facilities used in the provisioning of a service, network communications between the facilities for the purpose of backup, management, and workload communications are cryptographically protected in transit between data centre facilities.
```

**Review by walterkozlowski — APPROVED** [review]

File: doc/ref_model/chapters/chapter07.md (hunk 1 above)
Comment (karinesevilla) [reply doc/ref_model/chapters/chapter07.md]: @pgoyal01 Thanks, modifications committed!

**Comment by wmk-admin** [comment]
@petorre, @pgoyal01, @sukhdevkapur, @ASawwaf - can you guys review and approve the new structure if you can?

File: doc/ref_model/chapters/chapter07.md [hunk 2]
```diff
@@ -123,44 +123,46 @@ The platform supports the workload, and in effect controls access to the workloa
 <a name="7.4.1.2"></a>
 #### 7.4.1.2 The following general security requirements apply to the Cloud Infrastructure
 
-* Restrict traffic to (and from) the workload to only traffic that is necessary, and deny all other traffic.
+**System Hardening**
+* No login to root on any platform systems (platform systems are those that are associated with the platform and include systems that directly or indirectly affect the viability of the platform) when root privileges are not required.
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter07.md:127]: Suggest strengthening the language
```suggestion
* Adhering to the principle of least privilege, no login to root on any platform systems (platform systems are those that are associated with the platform and include systems that directly or indirectly affect the viability of the platform) when root privileges are not required.
```

File: doc/ref_model/chapters/chapter07.md [hunk 3]
```diff
@@ -123,44 +123,46 @@ The platform supports the workload, and in effect controls access to the workloa
...
+* Processes for managing platform access control filters must be documented, followed, and monitored.
 * Role-Based Access Control (RBAC) must apply for all platform systems access.
-* An approved system or process for last resort access must exist for the platform.
-* All API access must use TLS protocol.
+* All APIs access must use TLS protocol, including back-end APIs.
+
+**Workload security**
+* Restrict traffic to (and from) the workload to only traffic that is necessary, and deny all other traffic.
+* Support zoning within a tenant workload - using application-level filtering.
+* Not expose tenant IP address details to another tenant.
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter07.md:144]:
```suggestion
* Not expose tenant internal IP address details to another tenant.
```

**Review by ulikleber — APPROVED** [review]

**Review by pgoyal01 — APPROVED** [review]

**Pull request merged**